src/core/Tag/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { ChipProps } from "@material-ui/core";
+import ClearIcon from "@material-ui/icons/Clear";
+import React, { useState } from "react";
+import { StyledTag } from "./style";
+
+export interface SdsTagProps extends Omit<ChipProps, "color"> {
+  label: string;
+  sdsStyle?: "rounded" | "square";
+  sdsType?: "primary" | "secondary";
+  color?:
+    | "primary"
+    | "info"
+    | "success"
+    | "warning"
+    | "error"
+    | "gray"
+    | "beta"
+    | [string, string]
+    | [string, string, string];
+}
+
+type TagProps = SdsTagProps;
+
+export type { TagProps };
+
+const Tag = (props: SdsTagProps): JSX.Element => {
```

Comment:
Can we add MUI's component link here as a comment please? This way it's easier for people to find the underlying MUI component!

(This is a new convention SDS wants to add to all applicable components 👌 ) Thanks so much!

e.g.,

```tsx
/**
 * @see: https://v4.mui.com/components/chips/
 */
const Tag = (props: SdsTagProps): JSX.Element => {
```

Reply:
Excellent points. I'll update the branch accordingly :D